config/settings/production_aks.yml

```diff
@@ -0,0 +1,39 @@
+gcp_api_key: please_change_me
+publish_api_url: https://api3.publish-teacher-training-courses.service.gov.uk
+publish_url: https://www3.publish-teacher-training-courses.service.gov.uk
+find_url: https://www3.find-postgraduate-teacher-training.service.gov.uk
```

Comment:
They all have 3 in the urls, can you clarify please?

Reply:
See [previous comment](https://github.com/DFE-Digital/publish-teacher-training/pull/3692#discussion_r1257998016)